### PR TITLE
Feature/migrate comm site to mui5

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

File: packages/taraxa-ui/src/components/DataCard/DataCard.styles.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const useStyles = makeStyles(() => {
       backgroundColor: '#151823',
       paddingLeft: '24px',
       paddingRight: '24px',
-      border: '1px solid #15AC5B',
+      border: '1px solid #15AC5B !important',
```
Comment (Malak67) [inline packages/taraxa-ui/src/components/DataCard/DataCard.styles.ts:12]: Why do we need important here? Does it get overwritten somewhere else?

**Review by Malak67 — APPROVED** [review]
LGTM. Only concerned about using '!important'. we should try and avoid it as much as we can.

**Comment by farebord** [comment]
> LGTM. Only concerned about using '!important'. we should try and avoid it as much as we can.

I agree with this. Sometimes is difficult with MUI

**Review by farebord — APPROVED** [review]
Looks good to me! Great work @VargaElod23

File: packages/taraxa-ui/src/components/DataCard/DataCard.styles.ts (hunk 1 above)
Comment (VargaElod23) [reply packages/taraxa-ui/src/components/DataCard/DataCard.styles.ts:12]: Yes, it gets overridden for some reason by the `Card` component `root` styles.

**Pull request merged**